spec/datetimeformat.html

```diff
@@ -66,7 +66,7 @@ <h1>Abstract Operations For DateTimeFormat Objects</h1>
         <tr>
           <td>[[FractionalSecondDigits]]</td>
           <td>*"fractionalSecondDigits"*</td>
-          <td>*1*, *2*, *3*</td>
+          <td>*1*<sub>𝔽</sub>, *2*<sub>𝔽</sub>, *3*<sub>𝔽</sub></td>
```

Comment:
To match ES2021 we add the `<sub>𝔽</sub>` suffix for Number values, distinguishing them from BigInt